crates/re_viewer/src/ui/welcome_screen/welcome_page.rs

```diff
@@ -58,10 +58,39 @@ fn onboarding_content_ui(
                 Visualize synchronized data from multiple processes, locally or over a network.",
             image: &re_ui::icons::WELCOME_SCREEN_LIVE_DATA,
             add_buttons: Box::new(|ui: &mut egui::Ui| {
-                // TODO(ab): activate when C++ is ready!
-                // url_large_text_button(ui, "C++", CPP_QUICKSTART);
-                url_large_text_button(ui, "Python", PYTHON_QUICKSTART);
-                url_large_text_button(ui, "Rust", RUST_QUICKSTART);
+                if large_text_button(ui, "C++").clicked() {
+                    open_quick_start(
+                        command_sender,
+                        [
+                            include_str!("../../../data/quick_start_guides/cpp_native.md"),
+                            include_str!("../../../data/quick_start_guides/how_does_it_work.md"),
+                        ],
+                        "C++ Quick Start",
+                        "cpp_quick_start",
+                    );
+                }
+                if large_text_button(ui, "Python").clicked() {
+                    open_quick_start(
+                        command_sender,
+                        [
+                            include_str!("../../../data/quick_start_guides/python_native.md"),
+                            include_str!("../../../data/quick_start_guides/how_does_it_work.md"),
+                        ],
+                        "Python Quick Start",
+                        "python_quick_start",
+                    );
+                }
+                if large_text_button(ui, "Rust").clicked() {
+                    open_quick_start(
+                        command_sender,
+                        [
+                            include_str!("../../../data/quick_start_guides/rust_native.md"),
+                            include_str!("../../../data/quick_start_guides/how_does_it_work.md"),
+                        ],
+                        "Rust Quick Start",
+                        "rust_quick_start",
+                    );
+                }
```

Comment:
We should make sure that `main` is always in a good state, i.e. not link to stubbed files.

We can solve this with a simple feature flag (a hard-coded boolean works). That way we can still merge new feature (and have them checked by the compiler), yet still keep `main` in a releasable state (i.e. still link to `https://www.rerun.io/docs/getting-started/rust` until we've fleshed out all the markdown files)

https://trunkbaseddevelopment.com/feature-flags/

Reply:
This might be retro discussion material, but I feel this is somewhat in disagreement with our habit of fast-action PR/chained PR.

I'm literally going to work on this content next, so for now let's just keep that PR open until I do.

Reply:
Well, I have done that for the C++ guide now.